compiler-rt/lib/builtins/extendhfsf2.c

```diff
@@ -16,6 +16,10 @@ COMPILER_RT_ABI NOINLINE float __extendhfsf2(src_t a) {
   return __extendXfYf2__(a);
 }
 
+COMPILER_RT_ABI NOINLINE long double __extendhfxf2(src_t a) {
```

Comment:
I think this should be in a separate `extendhfxf2.c` file.  This file is for converting `half -> single`.

Reply:
`__extendXfYf2__` is defined in `fp_extend_impl.inc`, not in here.  You can define `SRC_HALF`, `DST_SINGLE`, and include `fp_extend_impl.inc` in that separate file.

Reply:
Ok. Sure.